Codeinwp/neve
# feat: expand locally loading of google fonts beyond just neve

### Summary
Code was moved to neve pro See: https://github.com/Codeinwp/neve-pro-addon/pull/1975

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


See: https://github.com/Codeinwp/neve-pro-addon/pull/1975


Closes https://github.com/Codeinwp/neve-pro-addon/issues/1961
